student.rb

```diff
@@ -0,0 +1,12 @@
+require './person'
+# Students' information/actions in the library.
+class Student < Person
+  def initialize(age, classroom, name = 'Unknown')
+    super(age, name)
```

Comment:
- [ ] [REQUIRED] Good job so far! In line 4, you should add `parent_permission` after `super(age, name)`!

#

**[HINT] It would like this ``super(age, name, parent_permission)``.**

Reply:
Hello @mukhammadrizooff why should I put this argument if it already exists on person.rb?